Kitodo/setup/migrate_index/add_new_index_1.11.x.sql

```diff
@@ -1,5 +1,5 @@
 --
--- Add new indexes for Goobi.Production CE newer than 1.11.2
+-- Add new indexes for Kitodo.Production CE newer than 1.11.2
```

Comment:
This comment should be renamend to "Add new indexes for Kitodo.Production 2.0.0" and the file name to "add_new_index_2.0.0.sql" as in 2.x branch.